### PR TITLE
Set timeout policy on requests

### PR DESCRIPTION
**Pull request by kislyuk**

### CODE REVIEW

File: hca/util/__init__.py [hunk 1]
```diff
@@ -107,6 +107,9 @@ def __init__(self, retry_after_status_codes={301}, **kwargs):
 
 
 class _ClientMethodFactory(object):
+    retry_policy = RetryPolicy(read=10, status=10, status_forcelist=frozenset({502, 503, 504}))
```
Comment (ttung) [inline hca/util/__init__.py]: Do you want to add 500? c.f., https://github.com/HumanCellAtlas/dcp-preview/issues/10

File: hca/util/__init__.py [hunk 2]
```diff
@@ -107,6 +107,9 @@ def __init__(self, retry_after_status_codes={301}, **kwargs):
...
+    timeout_policy = timeout.Timeout(connect=300, read=300)
```
Comment (ttung) [inline hca/util/__init__.py]: Those both seem really high to me.

File: hca/util/__init__.py (hunk 1 above)
Comment (kislyuk) [reply hca/util/__init__.py]: Great catch. Added.

File: hca/util/__init__.py (hunk 2 above)
Comment (kislyuk) [reply hca/util/__init__.py]: Adjusted to 60, 120.

**Comment by kislyuk** [comment]
Merging because https://travis-ci.org/HumanCellAtlas/dcp-cli/builds/361328725 (the pre-rebase version) succeeded & verbal approval.

**Pull request merged**